README.md

```diff
@@ -0,0 +1,146 @@
+<div align="center">
+  <h1><b>My Enumerable</b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 My Enumerable ](#-morse-code-translator-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites ](#prerequisites-)
+    - [Setup ](#setup-)
+    - [Install ](#install-)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My Enumerable <a name="about-project"></a>
+
+**My Enumerable**
+
+## 🛠 Built With <a name="built-with"></a>
+
+<!-- - **Ruby** -->
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <a href="https://www.ruby-lang.org/en/">Ruby</a>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Function to return boolean values depending on state of arguments and conditions.
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+- Get a copy of the project
+- Come into the project directory
+- Open the project using code editor of your choice
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project:
+ - Ruby installed
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/David-Lanzz/MyEnumerable.git
+```
+
+### Install <a name="install"></a>
+
+Install all the dependencies required for, by using the command below
+
+```sh
+  cd my-project
+  npm install
+```
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Muhammad-Usman**
+- GitHub: [@githubhandle](https://github.com/MuhammadUsmanAslam)
+- Twitter: [@twitterhandle](https://twitter.com/m_usman_aslam)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/muhammad-usman-aslam)
+
+👤 **David-Lanzz**
+- GitHub: [@githubhandle](https://github.com/David-Lanzz/)
+- Twitter: [@twitterhandle](https://twitter.com/LanzzDavid)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/david-lanzz)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **N/L**
```

Comment:
- [ ] Don't use abbreviation, be clear about 1-4 future features that you want to implement on this project so the users better unsterstand it